Common metadata updates (#712)
radiantearth/stac-spec: item-spec/common-metadata.md

# STAC Common Metadata

> This document outlines all commonly used fields that are currently supported in STAC Item properties. These fields are 

Reviewer: ```suggestion
This document outlines all commonly used fields for STAC Item properties. These fields are 
```

'that are currently supported' sounds a vit exclusive as if that are the only fields supported.

Various *examples* are available in the folder [`examples`](examples/).
*JSON Schemas* can be found in the folder [`json-schema`](json-schema/).

## Basics

Descriptive fields to give a basic overview of a STAC Item.

- [JSON Schema](json-schema/basics.json)

| Field Name  | Type   | Description                                                  |
| ----------- | ------ | ------------------------------------------------------------ |
| title       | string | A human readable title describing the Item. |

> | description | string | Detailed multi-line description to fully explain the Item. [CommonMark 0.29](https://commonmark.org/) syntax MAY be used for rich text representation. |

Reviewer: Adding the description field is an addition which should be put into the changelog.

Author: I added the description and a little bit more about the common metadata basics fields to the changelog, let me know if it's overkill

Reviewer: Some of the entries will probably be removed once the changelog is merged for 0.9 as these are no changes between 0.8.x and 0.9. But we can handle that later, I guess.
